frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -279,12 +280,12 @@ export const InstallPlanDetails: React.SFC<InstallPlanDetailsProps> = ({ obj })
                   </dd>
                 ))}
                 <dt>Catalog Sources</dt>
-                {(_.get(obj.status, 'catalogSources') || []).map((catalogName) => (
+                {_.get(obj, 'status.catalogSources', []).map((catalogName) => (
                   <dd key={catalogName}>
                     <ResourceLink
                       kind={referenceForModel(CatalogSourceModel)}
                       name={catalogName}
-                      namespace={obj.spec.sourceNamespace || olmNamespace}
+                      namespace={obj.spec.sourceNamespace || DEFAULT_SOURCE_NAMESPACE}
```

Comment:
I'm not sure if this is right for native k8s.

cc @alecmerdler

Reply:
Hmm, right. Do we need to set the default namespace based on a feature flag?

Reply:
After speaking with the OLM team, it sounds like there's no way to know the namespace. I think we have to display it as plain text if `spec.sourceNamespace` is not set rather than a link.